### ISSUE TITLE
`onlyShowOnTab` Autocomplete setting does not work in 2.7.4

### ISSUE THREAD

**Issue by gh-jir** [withfig/fig#1984]
### Checks

- [X] I have searched [github.com/withfig/fig/issues](https://github.com/withfig/fig/issues?q=) and there are no duplicates of my issue
- [X] I have run `fig doctor` in the affected terminal session
- [X] I have run `fig restart` and replicated the issue again

### Operating system

macOS 12.6.0 (21G115)

### Expected behaviour

Autocomplete appears as normal in Terminal.

### Actual behaviour

Autocomplete is missing from prompt.

![image](https://user-images.githubusercontent.com/97641518/199889841-b2deb486-7663-45ec-addd-2c64eb1edb9c.png)

### Environment

```yaml
fig-details:
  - 2.7.4
hardware-info:
  - model: 
  - model-id: 
  - chip-id: Apple M1 Pro
  - cores: 10
  - mem: 16.00 GB
os-info:
  - macOS 12.6.0 (21G115)
environment:
  - shell: /bin/zsh
  - terminal: iterm
  - cwd: /Users/jirwong/development/branches
  - exe-path: /opt/homebrew/bin/fig
  - install-method: brew
  - env-vars:
    - FIGTERM_SESSION_ID: bf4f0fe8-e7df-41fc-b53a-f0aeeb701f25
    - FIG_HOSTNAME: Jir-mac.local
    - FIG_TERM: 2.7.4
    - PATH: /Users/jirwong/.nodenv/shims:/Users/jirwong/Library/pnpm:/usr/local/sbin:/Library/Frameworks/Python.framework/Versions/2.7/bin:/Library/Frameworks/Python.framework/Versions/3.10/bin:/opt/homebrew/bin:/opt/homebrew/sbin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:~/bin:/Library/Apple/usr/bin:/Users/jirwong/.fig/bin:/Users/jirwong/.local/bin:/Users/jirwong/Library/Application Support/JetBrains/Toolbox/scripts
    - SHELL: /bin/zsh
    - TERM: xterm-256color
    - FIG_PID: 11197
    - FIG_PARENT: bf4f0fe8-e7df-41fc-b53a-f0aeeb701f25
    - FIG_SET_PARENT: bf4f0fe8-e7df-41fc-b53a-f0aeeb701f25
```

**Comment by gh-jir** [comment]
This was working yesterday but not sure why it's not working today anymore.

**Comment by mschrage** [comment]
@gh-jir can you check your internet connect and restart Fig?

**Comment by gh-jir** [comment]
Hey @mschrage thanks for the reply.

Yeah I've checked my internet connection, restarted Fig, and even reinstalled Fig several times.

Autocomplete's still missing from my prompt; I've tried it with several applications (eg Terminal, iTerm2) as well.

**Comment by SeparateRecords** [comment]
Can you go to the ◧ Fig menubar icon → Developer → Autocomplete Devtools → Console tab, then take a screenshot? That'll show whether fig failed to load for whatever reason

Also I know it's a PITA but does a full system restart fix?

**Comment by agronom81** [comment]
I have same error om my macOS 11.7
![Fig_2022-11-04 07-34-05](https://user-images.githubusercontent.com/3888562/199897858-65b299f4-4257-4b92-804d-b24d6d9dfbc9.jpg)

**Comment by gh-jir** [comment]
@SeparateRecords unfortunately no, full system restart didn't fix it.

Yeah @agronom81 I get the same console output as well.

![image](https://user-images.githubusercontent.com/97641518/199898359-d8e9a222-6a2a-4b36-a3ba-2dffe4d62ffa.png)

**Comment by SeparateRecords** [comment]
After a long debugging session on Discord we figured it out! Somehow there's been a regression with only show on tab. The fix was `fig settings --delete autocomplete.onlyShowOnTab`

@agronom81 do you have this setting enabled? Can you try deleting it?

**Comment by grant0417** [comment]
Oh no! Regression with `onlyShowOnTab` seems to be because we are not intercepting tab by default any more, will try to get a fix to beta tomorrow (11/4)

**Comment by dr2009** [comment]
open Autocomplete Devtools
then clear cache works for me (choose newwork tab, click the right button)

**Comment by agronom81** [comment]
> After a long debugging session on Discord we figured it out! Somehow there's been a regression with only show on tab. The fix was `fig settings --delete autocomplete.onlyShowOnTab`
> 
> @agronom81 do you have this setting enabled? Can you try deleting it?

the @dr2009's  decision helped

**Comment by imanAzi** [comment]
I have the same issue:
![image](https://user-images.githubusercontent.com/25829454/199943857-07f9e63d-90d3-480a-9d4a-93af921231b2.png)

I both tried: 

```
fig settings --delete autocomplete.onlyShowOnTab
error: No settings found matching autocomplete.onlyShowOnTab
```

And cleared cache via Autocomplete Devtools. 

When putting a command in my shell I see a line appearing under the command but nothing more than that:
![image](https://user-images.githubusercontent.com/25829454/199944899-5f353d85-0158-4aac-abf3-b454263628ac.png)

**Comment by mschrage** [comment]
@imanAzi can you run `fig issue` - it seems like you are running into a different bug.

**Comment by imanAzi** [comment]
@mschrage yes both full screen and not in full screen it doesn't auto-complete, I get the following screen:
![image](https://user-images.githubusercontent.com/25829454/200080052-ad80f85f-08fe-4458-a10d-1e37cd10b8e0.png)

Where after my first command a line appears under the command, I don't see the suggestions but if I move around with the arrow keys and press 'Enter' it selects one of the suggestions that normally should be shown.

**Comment by grant0417** [comment]
The `onlyShowOnTab` issues should be fixed on beta `2.7.5-beta.0`

**Comment by gh-jir** [comment]
Works great for me, awesome stuff! 🥇
